scripts/utils.js

```diff
@@ -1,21 +1,48 @@
-const path = require('path');
+const path = require('node:path');
 
+const { addresses, Pool } = require('@nexusmutual/deployments');
+const ethers = require('ethers');
+
+/**
+ * Retrieves the latest on-chain coverAssets IDs to their corresponding token symbols mapping.
+ *
+ * @param {ethers.providers.Provider} provider - The Ethereum provider to interact with the blockchain.
+ * @returns {Promise<Object>} A promise that resolves to an object mapping cover asset IDs to their symbols.
+ */
+const getCoverAssetsMap = async provider => {
+  const BASE_COVER_ASSETS_MAP = { 0: 'ETH', 255: 'NXM' };
+
+  const pool = new ethers.Contract(addresses.Pool, Pool, provider);
+  const assets = await pool.getAssets();
+
+  // Filter for cover assets and exclude ETH address, but keep track of original index
+  const coverAssets = assets
+    .map((asset, index) => ({ ...asset, assetId: index }))
+    .filter(([address, isCoverAsset]) => isCoverAsset && address !== '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE');
+
+  // Get symbol for each cover asset and use original index as key
+  for (const { 0: address, assetId } of coverAssets) {
```

Comment:
`const { 0: address, assetId }` what's this? :raised_eyebrow:

Reply:
fixed. you caught me in the middle of refactoring.